### PR TITLE
docs: case fixes

### PR DESCRIPTION
**Pull request by padcom**
A few upper/lower case changes, probably stemming from previous versions of the document.

### CODE REVIEW

File: docs/rfcs/data-loaders/index.md [hunk 1]
```diff
@@ -810,7 +810,7 @@ TBD: is this worth it? Are any other functions needed?
 
 ### Suspense
 
-Using Suspense is probably the first alternative that comes to mind and it has been considered as a solution for data fetching by implementing proofs of concepts. It however suffers from major drawbacks that are tied to its current design and is not a viable solution for data fetching.
+Using `Suspense` is probably the first alternative that comes to mind and it has been considered as a solution for data fetching by implementing proofs of concepts. It however suffers from major drawbacks that are tied to its current design and is not a viable solution for data fetching.
```
Comment (posva) [inline docs/rfcs/data-loaders/index.md:813]:
```suggestion
Using Suspense is probably the first alternative that comes to mind and it has been considered as a solution for data fetching by implementing proofs of concepts. It however suffers from major drawbacks that are tied to its current design and is not a viable solution for data fetching.
```

**Review by posva — CHANGES_REQUESTED** [review]
Thanks a lot! let's keep Data Loaders with both uppercase

**Comment by padcom** [comment]
> Thanks a lot! let's keep Data Loaders with both uppercase

Sure - how about consistency? At the beginning of the document, those are mostly Upper Case, then the closer to the end they are all lower case. Are you saying you want to keep them throughout the document as Data Loaders, both uppercased?

**Comment by posva** [comment]
Not necessarily. Let’s just revert the one that were changed. I will deal with consistency later on

**Comment by padcom** [comment]
Done.

**Review by posva — APPROVED** [review]
Thanks a lot!

**Pull request merged**